kie-wb/kie-wb-webapp/pom.xml

```diff
@@ -1504,7 +1502,7 @@
             <server>org.jboss.errai.cdi.server.gwt.EmbeddedWildFlyLauncher</server>
             <disableCastChecking>true</disableCastChecking>
             <runTarget>kie-wb.html</runTarget>
-            <hostedWebapp>src/main/webapp</hostedWebapp>
+            <hostedWebapp>${project.build.directory}/kie-wb.war</hostedWebapp>
```

Comment:
src/main/webapp is part of sources and thus stored in git and that's the reason why I did copy it to target instead. Note that there are bunch of staff copied there during the build. Anyway not sure why src/main/webapp is used for this purpose so can't say for sure...